src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/grouped/DeferredListByCourtReportITest.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.hmcts.juror.api.moj.report.grouped;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupByResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupedReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupedTableData;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractGroupedReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.util.List;
+
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/administration/createUsers.sql",
+    "/db/mod/reports/DeferredListByCourtReportITest_typical.sql"
+})
+@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")//False positive
+class DeferredListByCourtReportITest extends AbstractGroupedReportControllerITest {
+
+    @Autowired
+    public DeferredListByCourtReportITest(TestRestTemplate template) {
+        super(template, DeferredListByCourtReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getCourtJwt("415");
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .build());
+    }
+
+    @Test
+    void positiveTypicalNotFound() {
+        testBuilder()
+            .jwt(getCourtJwt("475"))
+            .triggerValid()
+            .responseConsumer(this::verifyAndRemoveReportCreated)
+            .assertEquals(createResponse(0, new GroupedTableData()));
+    }
+
+    @Test
+    void positiveTypical() {
+        testBuilder()
+            .triggerValid()
+            .responseConsumer(this::verifyAndRemoveReportCreated)
+            .assertEquals(getTypicalResponse());
+    }
```

Comment:
Need a bureau flow (should see multiple courts)

Reply:
Added